src/main/java/com/eleks/academy/whoami/core/state/ProcessingQuestion.java

```diff
@@ -39,13 +43,39 @@ public void ask(String player, PlayerAction.Action question) {
 		throw new GameException("Not implemented");
 	}
 
-	;
+	public void answer(String player, String value) {
+		if (turn.getCurrentTurn().get(0).getValue() != null) {
+			if (isAnswerState()) {
+				turn.action(player, value);
+				answers.add(value);
+				if (answers.size() == players.size() - 1) {
```

Comment:
you may create in turn method `boolean hasTurnEnded()` to know if all players answered